Misc/NEWS.d/next/Core and Builtins/2020-06-30-20-17-31.bpo-41175.acJoXB.rst

```diff
@@ -0,0 +1 @@
+Guard against a possible NULL pointer dereference within bytearrayobject.
```

Comment:
Would you mind to clarify which operation is affected? I understood that it's: bytearray+bytearray.

Reply:
Amended.